tests/acceptance/routing-test.js

```diff
@@ -15,87 +15,51 @@ module('Acceptance | routing', {
-
-  const setupTest = create_file("initializers.setup-test.js",
-    "import Ember from 'ember';\n\nexport default {\n  name: 'setup-test',\n  initialize: function(container, app) {\n    app.setupForTesting();\n     app.injectTestHelpers();\n    window.QUnit = window.parent.QUnit;\n  }\n};");
-
-  let files = {};
-  [
-    appTemplate,
-    aboutTemplate,
-    indexTemplate,
-    router,
-    twiddleJson,
-    setupTest
-  ].forEach(function(file) {
-    files[file.filename] = file;
-  });
-
-  const owner = server.create('owner', {login: login});
-  server.create('gist', {
-    id: gist_id,
-    owner: owner,
-    files: files
-  });
+  const files = [
+    {
+      filename: "about.template.hbs",
+      content: "<p>About Page</p>"
+    },
+    {
+      filename: "index.template.hbs",
+      content: "<p>Main Page</p>"
+    },
+    {
+      filename: "application.template.hbs",
+      content: "{{#link-to \"index\" class=\"test-index-link\"}}Index{{/link-to}}\n{{#link-to \"about\" class=\"test-about-link\"}}About{{/link-to}}\n\n{{outlet}}"
+    },
+    {
+      filename: "router.js",
+      content: "import Ember from 'ember';\nimport config from './config/environment';\n\nvar Router = Ember.Router.extend({\n  location: config.locationType\n});\n\nRouter.map(function() {\n  this.route(\"about\");\n});\n\nexport default Router;\n"
+    },
+    {
+      filename: "twiddle.json",
+      content: "{\n  \"version\": \"0.4.0\",\n  \"dependencies\": {\n    \"jquery\": \"https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.js\",\n    \"ember\": \"https://cdnjs.cloudflare.com/ajax/libs/ember.js/1.13.5/ember.js\",\n    \"ember-data\": \"https://cdnjs.cloudflare.com/ajax/libs/ember-data.js/1.13.5/ember-data.js\"\n  }\n}"
+    }
+  ];
 
   const iframe = '#dummy-content-iframe';
-  const aboutLink = '.ember-application .test-about-link';
-  const indexLink = '.ember-application .test-index-link';
-  const outletText = '.ember-application p';
+  const aboutLink = '.test-about-link';
+  const indexLink = '.test-index-link';
+  const outletText = 'p';
   let iframe_window;
 
-  visit('/35de43cb81fc35ddffb2');
+  runGist(files);
 
   andThen(function() {
     iframe_window = find(iframe)[0].contentWindow;
 
-    // Wait until iframe loads
-    return new Ember.RSVP.Promise(function (resolve) {
-      iframe_window.addEventListener('load', function () {
-        iframe_window.removeEventListener('load');
-        return resolve();
-      });
-    });
+    iframe_window.click(iframe_window.find(aboutLink));
   });
 
   andThen(function() {
-    iframe_window.visit('/');
-  });
+    assert.equal(iframe_window.find(outletText).text().trim(), 'About Page', 'About Link leads to About Page being displayed');
```

Comment:
This is repeated a few times, can it be extracted to a test helper?

Something like:

```javascript
const iframe = '#dummy-content-iframe';

Ember.Test.registerHelper('outputPane', function(app) {
  return find(iframe)[0].contentWindow;
});

Ember.Test.registerHelper('outputContents', function(app, selector) {
  let output = app.testHelpers.outputPane();
  return output.find(selector).text().trim();
});

// then you can use them as follows:
outputPane().click(aboutLink);
andThen(() => {
  assert.equal(outputContents(outletText), 'About Page');
});
```